src/routes/Counter/modules/counter.js

```diff
@@ -47,7 +47,7 @@ const ACTION_HANDLERS = {
 // ------------------------------------
 // Reducer
 // ------------------------------------
-const initialState = 0
+const initialState = 5
```

Comment:
Because from a UX perspective clicking on Double Async before incrementing appears to do nothing - you cant double 0!